tests/test_transform.py

```diff
@@ -400,11 +513,107 @@ def checkTransformFromFrameSet(self, fromName, toName):
 
                     self.checkTransformation(permTransform, mapping=polyMap, msg=msg)
 
+    def checkAllGetInverse(self, fromName, toName):
```

Comment:
A nicely written method, but what does the "All" in the name tell me? Why not "checkGetInverse"?